exercises/04_count_rune/main_test.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "testing"
+
+func TestCountRune(t *testing.T) {
+	for _, tc := range []struct {
+		name   string
+		input  string
+		inputr rune
+		want   int
+	}{
+		{"empty", "", 101, 0},
```

Comment:
The same as with counting bytes - you don't probably remember what ascii 101 is (-; `'e'` is also a constant in Go